zuul.d/project.yaml

```diff
@@ -38,4 +38,5 @@
     templates:
       - ansible-collection-kubernetes-molecule-focal
       - ansible-collection-kubernetes-molecule-jammy
+      - ansible-collection-kubernetes-molecule-noble
```

Comment:
This template doesn't exist, you'll have to define it, but I would also much rather just use GHA for the testing here, I think you can delete the project template definitions in this case, and then delete the templates from the job here.